packages/react/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -400,6 +400,25 @@ export const DatePickerBase: React.FunctionComponent<IDatePickerProps> = React.f
     );
   };
 
+  const onRenderReadOnlyInput = (inputProps: ITextFieldProps, inputIds: ITextFieldInputIds): JSX.Element | null => {
```

Comment:
Nit: Should this be renderReadOnlyInput?  It doesn't feel like an event handler.